amenity-points.mss

```diff
@@ -9,6 +9,7 @@
 @airtransport: #8461C4; //also ferry_terminal
 @health-color: #da0092;
 @amenity-brown: #734a08;
+@gastronomy: @amenity-brown;
```

Comment:
Could we just call it `@food`?

Reply:
This was my first idea, but it's also about drinking (bar or pub).

Reply:
I'd still rather the shorter and less precise name

Reply:
I like the more precise name, but that's not the most important thing here. What others think about it?

Reply:
gastronomy has less confusion potential

Reply:
+1 for gastronomy